src/Controller/Component/PrgComponent.php

```diff
@@ -2,26 +2,70 @@
 namespace Search\Controller\Component;
 
 use Cake\Controller\Component;
-use Cake\Event\Event;
 
 class PrgComponent extends Component
 {
+
+    /**
+     * Default config
```

Comment:
Yes, I agree with that. This "WTF?" happened to me as well when I started using the plugin. IMHO you should have to explicit set the actions instead of the component processing all by default.